lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -378,6 +383,9 @@ def setUp(self):
     def test_create_thread(self, mock_request):
         self.create_thread_helper(mock_request)
 
+    def test_create_thread_with_context(self, mock_request):
```

Comment:
I'd suggest that you allow the context to be passed in (can make the default value standalone if you wish).